src/components/exchange/DirectExchangeSelection.tsx

```diff
@@ -34,6 +34,7 @@ export function DirectExchangeSelection({
     const [value, setValue] = useState<string>("");
     const [selectedClass, setSelectedClass] = useState<string>("");
     const [student, setStudent] = useState<string>("");
+    const [include, setInclude] = useState<boolean>(false);
```

Comment:
Please choose a better name for this variable, as just saying `include` is extremely ambiguous.

For example, `isExchangeSelectionIncluded`